### PR TITLE
Include triggers in the table migration

### LINKED ISSUE #16: Add support to migrate mysql triggers

### PR DESCRIPTION
**Pull request by martinmca**
Extract trigger information from the source table and re-create the trigger(s) on the destination table.
Fixes #16.

### CODE REVIEW

**Comment by michaeljoseph** [comment]
@martinmca looks like it's just linting failing the build: https://travis-ci.org/MapleSyrupGroup/dbsampler/jobs/624177666?utm_medium=notification&utm_source=github_status

File: src/Migrator.php [hunk 1]
```diff
@@ -228,12 +228,14 @@ private function ensureEmptyTargetTable($table, Connection $sourceConnection, Co
...
         $destConnection->exec('DROP TABLE IF EXISTS ' . $sourceConnection->quoteIdentifier($table));
-
+        $triggerSql = [];
         $driverName = $sourceConnection->getDriver()->getName();
         if ($driverName === 'pdo_mysql') {
             $createSqlRow = $sourceConnection->query('SHOW CREATE TABLE ' . $sourceConnection->quoteIdentifier($table))
                 ->fetch(\PDO::FETCH_ASSOC);
             $createSql = $createSqlRow['Create Table'];
+            // find any triggers
+            $triggerSql = $this->checkTableTriggers($table, $sourceConnection);
```
Comment (michaeljoseph) [inline src/Migrator.php]: Not sure `check` is such a great verb here, since it's returning / creating the triggers?

File: src/Migrator.php [hunk 2]
```diff
@@ -228,12 +228,14 @@ private function ensureEmptyTargetTable($table, Connection $sourceConnection, Co
...
         // SchemaManager doesn't do enums!
         $destConnection->exec('DROP TABLE IF EXISTS ' . $sourceConnection->quoteIdentifier($table));
-
+        $triggerSql = [];
         $driverName = $sourceConnection->getDriver()->getName();
         if ($driverName === 'pdo_mysql') {
             $createSqlRow = $sourceConnection->query('SHOW CREATE TABLE ' . $sourceConnection->quoteIdentifier($table))
                 ->fetch(\PDO::FETCH_ASSOC);
             $createSql = $createSqlRow['Create Table'];
+            // find any triggers
```
Comment (michaeljoseph) [inline src/Migrator.php]: This comment can probably go if we have a better function name?

File: src/Migrator.php (hunk 1 above)
Comment (martinmca) [reply src/Migrator.php]: I have changed to the function name to be more reflective of the functions functionality

**Comment by michaeljoseph** [comment]
@martinmca Shall we try pinning to the version of `php-cs-fixer` in the last green build?
`friendsofphp/php-cs-fixer (v2.13.1)`
https://travis-ci.org/MapleSyrupGroup/dbsampler/jobs/456941537

**Comment by martinmca** [comment]
Updated to v2.16.0 as the highest php-cs-fixer version that still passes the tests.

**Review by michaeljoseph — APPROVED** [review]
🚀

**Review by noondaysun — APPROVED** [review]

**Pull request merged**